enterprise/websvc.manager/external/binaries-list

```diff
@@ -0,0 +1,17 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+8531AD5AC454CC2DEB9D4D32C40C4D7451939B5D javax.xml.bind:jaxb-api:2.3.1
```

Comment:
I have a feeling this JAR has already been referenced before. We don't want to duplicate JARs. A special `org.netbeans.libs.xml.bind` wrapper module should rather be created.

Reply:
We have `xml.jaxb.api` and `libs.jaxb` in the ide cluster. If version 2.2.X is good enough, that might be an alternative.

Reply:
had added module xml.jaxb.api as dependency and it worked